### ISSUE TITLE
[REQUEST] Tagged Releases for the Frontend

### ISSUE THREAD

**Issue by abenson** [projectM-visualizer/projectm#834]
### Please confirm the following points:

- [X] This question is NOT about the Android apps in the Play Store
- [X] I have [searched the project page](https://github.com/search?q=org%3AprojectM-visualizer+type%3Aissue+&type=issues) to check if the question was already asked elsewhere

### Topic

General Request

### Your Request

With the split of projectM into libprojectM and a separate frontend, the frontend is no longer packaged as part of projectM.

For Void Linux, this means we will not have the frontend if we just upgrade to projectM 4.x. The frontend does not currently have tagged releases, and therefore does not yet meet the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements) for inclusion into our repository.

> Software need to be used in version announced by authors as ready to use by the general public - usually called releases. Betas, arbitrary VCS revisions, templates using tip of development branch taken at build time and releases created by the package maintainer won't be accepted.

If the goal is to mature the project before beginning to tag releases, that's fine.  I appreciate your time! Thank you.

**Comment by kblaschke** [comment]
Yes, this is actually what prevents us from "spreading the word" across more distros. The SDL2 frontend is still in active development, and while already quite usable, not fully finished, especially the packaging part, which is most important for a general-public release.

I'll move this issue over to the frontend-sdl2 repo and keep it open. Whenever a final (non-beta) release is ready, I'll post a notice here!
